README.md

```diff
@@ -18,7 +18,7 @@ A comparison with rust-cpython can be found [in the guide](https://pyo3.rs/maste
 
 ## Usage
 
-PyO3 supports Python 3.6 and up. The minimum required Rust version is 1.45.0.
+PyO3 supports Python 3.6 and up. The minimum required Rust version is 1.41.1.
```

Comment:
Imo patch releases probably aren't that interesting unless we do actually depend on the patch fix. Shall we just use `major.minor` here?

```suggestion
PyO3 supports Python 3.6 and up. The minimum required Rust version is 1.41.
```

Reply:
Agreed.